### PR TITLE
Feature/full website

### PR DESCRIPTION
**Pull request by John-Kibirige**
### Activities worked on 
- Install react-router-dom via npm
- setup the navbar using react routing 
- add styles for the entire project 
- fix linter errrors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+import SharedComponent from './components/SharedComponent';
 
 function App() {
   return (
     <div>
-      <Calculator />
+      <BrowserRouter>
+        <Routes>
+          <Route path='/' element={<SharedComponent />}>
+            <Route path='home' element={<Home />} />
```
Comment (FrederickMih) [inline src/App.js]: - [ ] Hi @John-Kibirige, I think your Home page should be the root route. Right now `http://localhost:3000/` display a blank page which does not give a good user experience. Let's fix this issue.

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,23 @@
+/* eslint-disable  */
```
Comment (FrederickMih) [inline src/App.js]: - [ ] Please do not disable Linters. These are tools that help to flag programming bugs. Let's fix these errors.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @John-Kibirige , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Great commit history :heavy_check_mark: 
- PR has a good title and summary :+1: 
- Correct GitHub workflow :+1: 
- No Linter error :heavy_check_mark: 
- Website consist of the requested three pages :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @John-Kibirige,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**